GeometryService/src/TrackerMaker.cc

```diff
@@ -412,43 +412,50 @@ namespace mu2e {
     computeManifoldEdgeExcessSpace();
     // fill their content
     makeStraws(allStraws);
+    // create an empty G4Tracker: this gets filled further down
+    auto g4trackerptr = shared_ptr<G4Tracker>(new G4Tracker);
+    // see which planes actually exist.  This is deprecated, TrackerStatus should be used instead  FIXME!
+    std::array<bool,StrawId::_nplanes> planeExists;
+    for ( int ipln=0; ipln<StrawId::_nplanes; ++ipln ){
+      planeExists[ipln] =  ( find ( _nonExistingPlanes.begin(), _nonExistingPlanes.end(), ipln) == _nonExistingPlanes.end() );
```

Comment:
I will ask a question here that is for possible future work - it should not delay this PR.  Can you remind me of the plan looking forward for the functionality of non existing planes.  I remember a discussion between you and Ray on a previous PR - is the plan that it eventually moves to Proditions or to keep it here?